rosdep/python.yaml

```diff
@@ -4391,6 +4391,7 @@ python-ruamel.yaml:
   fedora: [python-ruamel-yaml]
   nixos: [pythonPackages.ruamel_yaml]
   ubuntu:
+    '*': [python-ruamel.yaml]
```

Comment:
Since it's not available on focal, is there a replacement package that should be used instead?